config/pipelines/high_throughput_heron_384_tailed.yml

```diff
@@ -2,6 +2,7 @@
 # This pipeline can be deleted once all "limber_heron_lthr" requests have progressed through it
 # It is being replaced by high_throughput_heron_384_tailed_v2
 Heron-384 Tailed A: # Heron 384-well pipeline specific to PCR 1 plate
+  pipeline_group: Heron-384
```

Comment:
I would call this `Heron-384 (legacy)` or something to make clear it's not used.